docs/application/dotnet/tutorials/test/testing-your-app-on-gear.md

```diff
@@ -0,0 +1,72 @@
+# Test Tizen .NET Application on a Gear device
+
+To deploy Tizen .NET Application to Samsung Gallaxy app store, you need to test it on Samsung Gear devices.
+As Samsung Gear doesn't have a USB port, Tizen provides a way to do it over Wi-Fi.
+
+Therefore, the Gear device should be connected to Wi-Fi, and the host PC to Wi-Fi or UTP cable to the same local network.
+
+![Host PC and the gear connect in local network](media/gear_wifi_connect.png)
+
+## Prepare The Gear Device
+
+To debug on the Samsung Gear device, use the following steps:
+
+1. Open the Settings app
+
+2. Scroll to the bottom of the menu and tap About watch.
+![About Watch](media/testing_your_app_on_gear1.png)
+
+3. Turn on Debugging mode.
```

Comment:
Suggestion:
Tap Debugging to turn on.

Please verify whether the technical meaning is altered.

Reply:
done